public/index.html

```diff
@@ -570,146 +574,156 @@
+          <p>Visit <a target="_blank" rel="noreferrer noopener" href="https://getusppe.org/give">getusppe.org</a> to let them know what you have available.</p>
 
-        <hr/>
+          <div class="question i18n" data-i18n="ftm-add-donation-site-question">Want to add a donation site? update or remove information?</div>
+          <p class="i18n" data-i18n="[html]ftm-add-donation-site-answer">Please <a href="https://docs.google.com/forms/d/e/1FAIpQLSfgCpK5coPVFC6rJrE7ZhimiZuDoEaL6fo6gYqxsN_FIpJZhg/viewform">fill out this form</a>. We will review it as soon as possible.</p>
+          <hr/>
 
-      </div>
-      <h2 class="i18n" data-i18n= "ftm-map-of-sites">Donation Sites</h2>
-      <div id="list-stats"></div>
+        </div>
+        <h2 class="i18n" data-i18n= "ftm-map-of-sites">Donation Sites</h2>
+        <div id="list-stats"></div>
 
-      <div class="locations-loading">
-        <img src="images/icons/loader.svg" alt="" class="load-spinner"> <span class="i18n" data-i18n="ftm-donations-site-loading">Loading donation sites...</span>
-      </div>
+        <div class="locations-loading">
+          <img src="images/icons/loader.svg" alt="" class="load-spinner"> <span class="i18n" data-i18n="ftm-donations-site-loading">Loading donation sites...</span>
+        </div>
 
-    <div class="locations-container" style="display: none;">
-      <div class="row" style="margin: 30px 0px 20px 0px;">
-        <div class="offset-xs-1 col-xs-10 offset-lg-2 col-lg-8">
-          <div class="map-search-wrap">
-            <div class="search-input">
-              <input type="search" name="map-search" id="map-search" placeholder="Search by city, zip, or address" data-i18n="[placeholder]ftm-maps-search-placeholder">
-              <!-- Icon credit to https://feathericons.com/  -->
-              <img class="search-icon" src="images/icons/search.svg">
+      <div class="locations-container" style="display: none;">
+        <div class="row" style="margin: 30px 0px 20px 0px;">
+          <div class="offset-xs-1 col-xs-10 offset-lg-2 col-lg-8">
+            <div class="map-search-wrap">
+              <div class="search-input">
+                <input type="search" name="map-search" id="map-search" placeholder="Search by city, zip, or address" data-i18n="[placeholder]ftm-maps-search-placeholder">
+                <!-- Icon credit to https://feathericons.com/  -->
+                <img class="search-icon" src="images/icons/search.svg">
+              </div>
+              <!-- BETA: To enable search field, either change data-enable-search to "true" in #map below, or add hide-search=false/show-search=true to query string. -->
+              <div class="search-controls">
+                <a href="javascript:;" class="map-action-link" id="use-location"><span data-i18n="ftm-maps-search-use-location"></span>
+                  Use my location<span class="search-control-separator">|</span></a>
```

Comment:
Span is a little off here

![image](https://user-images.githubusercontent.com/5788989/77830823-9d870900-7101-11ea-996c-de7c9e676e64.png)

Reply:
yep, looks like it